specifications/drtm-handler-and-table.rst

```diff
@@ -0,0 +1,276 @@
+DLE Handler Specification
+=========================
+
+This specification details the state the platform should be in when invoking a DLE Handler.
+
+Overview
+--------
+
+The DLE (Dynamic Launch Event) Handler consumes the TrenchBoot DRTM Table and invokes the platform's dynamic launch event. This specification seeks to standardize the invocation interface for the DLE Handler for each platform supported by TrenchBoot. This will provide a well-defined interface for DLE Handler implementors to follow and bootloader to comply to ensure interoperability between implementations.
```

Comment:
80 char wrap? It would be way easier to comment and suggest improvements.

Reply:
Ah, yes I should have done that!